pandas/core/frame.py

```diff
@@ -728,11 +728,35 @@ def iteritems(self):
         """
         Iterator over (column name, Series) pairs.
 
+        Iterates over columns as key, value dict-like pairs with columns name as keys and Series as values.
+
+        Returns
+        -------
+        it : generator
+            A generator that iterates over the columns of the frame.
+
         See also
         --------
         iterrows : Iterate over DataFrame rows as (index, Series) pairs.
         itertuples : Iterate over DataFrame rows as namedtuples of the values.
 
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [0.1, 0.2]},
+        ...                   index=['a', 'b'])
```

Comment:
Can you use something that looks more like a real world example? I think it makes things easier to read.

See for example the example DataFrame in https://pandas-docs.github.io/pandas-docs-travis/generated/pandas.DataFrame.reset_index.html